### DISCUSSION TITLE
Rebase checkbox never works

### DISCUSSION THREAD

**Discussion by TheDutchCoder** [renovatebot/renovate#11624] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

Checking the rebase checkbox never works for us. I tried checking it multiple times with and without hard-refreshing in between, it just never seems to pick it up.

Dependabot had this nice feature where you just comment on the PR and it would trigger a rebase, that might be a better solution.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by TheDutchCoder** [comment]
Additionally: rewriting the PR title to start with "rebase!" also does nothing. I don't think the bot watches the PR's for those things at all?

**Reply by viceice** [reply]
check the logs https://app.renovatebot.com/dashboard

**Reply by TheDutchCoder** [reply]
Oh so it _does_ run, it just doesn't re-run GH actions?

**Reply by rarkins** [reply]
What do you mean by "re-run"? Renovate doesn't "run" GH actions ever

**Reply by TheDutchCoder** [reply]
Sorry I wasn't being clear. Renovate rebasing should trigger GH actions, but for some reason it doesn't.
I'm wondering if (in this case) it's due to it not actually touching the lock file?

**Reply by rarkins** [reply]
A rebase request like that should result in a git force push. You should be able to confirm this in the PR history or by the commit timestamp. This force push will happen regardless of whether the contents have changed compared to the previous commit it is pushing over the top of. Whether such a push triggers GitHub Actions is outside the scope of Renovate.

**Reply by TheDutchCoder** [reply]
Okay fair enough.
My assumption is that the force push didn't actually change anything and GH actions therefore didn't rerun.

My other opened issue seems to be the real problem (lock file not updating). So I think we can end this discussion as renovate seems to be re-running, but there's just no visual feedback like with dependabot.

**Reply by rarkins** [reply]
> there's just no visual feedback like with dependabot.

Incorrect. You will see:
- A force push notification in the PR feed
- The checkbox you ticked becomes unticked
